src/components/Input/index.tsx

```diff
@@ -3,9 +3,11 @@ import { View, KeyboardTypeOptions } from 'react-native';
 import PropTypes from 'prop-types';
 import RenderIf from '../RenderIf';
 import StyledInput from './styled/input';
-import { Label, Error } from './styled/index';
+import { Label, Error, Icon } from './styled/index';
 import { BaseProps } from '../types';
 
+type InputIconPostions = 'left' | 'right';
```

Comment:
In src/components/types.d.ts there is a type:
```export type ButtonIconPosition = 'left' | 'right';```
change its name to IconPosition and use it here instead of defining it again, also change ButtonIconPosition to IconPosition where it is being used.

Reply:
Done